### PR TITLE
Created responsive hamburger menu

### PR DESCRIPTION
**Pull request by Gleennkar**
##Created the Responsive Hamburger Menu
- Added index.js file in order to give the hamburger menu its functionality 
- Link the various parts of my webpage
- Check for linter errors
- Used the figma template 1 as a reference

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,20 +10,33 @@
...
-            </ul>
-        </nav>
-    </header>
-
-    <section class="headline">
+             <div class="menu">
+            
+                <div id="myNav" class="overlay">
+              
+                      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
```
Comment (victorokoroji) [inline index.html:18]: - [x] You have done so well in implementing the mobile menu functionality 👍 . To improve your website user experience, I think you should increase the size and visibility of the` times` icon to match your chosen Figma template.  it is currently small and faint, making it difficult for users with eye impairment to read. 

| Figma Design | Your Design |
| ------------- | ------------- |
| image  | image  |

Comment (victorokoroji) [inline index.html:18]: - [x] Kindly ensure that the padding and margin applied to your times' icon matches your chosen Figma template. Lets give our users a good user experience 😄 

| Figma Design | Your Design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 2]
```diff
@@ -10,20 +10,33 @@
...
+              
+                  <div class="overlay-content">
+                      <a href="#Portfolio">Portfolio</a>
+                      <a href="#About">About</a>
+                      <a href="#Contact">Contact</a>
+                  </div>
```
Comment (victorokoroji) [inline index.html]: [OPTIONAL]

- [x] I would recommend you apply the same function that you used on the **times icon** to close the mobile menu on each of the menus(`onclick="closeNav()`).  Relying on HTML attributes might not be so effective 😄 .

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Gleennkar , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -10,20 +10,33 @@
...
+               </div>
+              
+              <nav>
+               <span onclick="openNav()"><img src="./Images/Union.png"  alt="hamburger"></span>
+              
+               <ul class="desktop-links">
+                   <li><a href="#Portfolio">Portfolio</a></li>
```
Comment (codecaiine) [inline index.html]: - [ ] [OPTIONAL] Good job on implementing the previous reviewer's requested changes :+1: however I would like to recommend you to work on this action when the user clicks (or taps) any option of the mobile menu (Example: **About or Portfolio or Contact** ) the mobile menu should disappear but in your case not. Please fix it in order to give your user a great **UX(User Experience )** :smile: 

Screenshot from 2022-07-18 20-07-57

**Review by codecaiine — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Clean code :heavy_check_mark:  
- Great UI :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**